x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -348,10 +347,9 @@ export const useQueryTimelineById = () => {
     } else {
       return Promise.resolve(resolveTimeline(timelineId))
         .then((result) => {
-          const data: ResolvedTimeline | null = getOr(null, 'data', result);
-          if (!data) return;
+          if (!result) return;
 
-          const timelineToOpen = omitTypenameInTimeline(data.timeline);
+          const timelineToOpen = omitTypenameInTimeline(result.timeline);
```

Comment:
we don't need the `omitTypenameInTimeline` anymore as typename is a relic of graphql. We may want to remove this and add a note that timeline imports from 7.x will require removing `typename` from the ndjson to import properly, but not sure how likely this is to happen. We could also just leave it to not overcomplicate this. Fine with whichever you decide here..

Reply:
Let's remove it while we're here :) https://github.com/elastic/kibana/pull/200633/commits/355510cdf1b22eca19deda68bbc8986765a21168

Reply:
Wonderful, thank you 👍🏾